tests/test_config.py

```diff
@@ -119,6 +119,26 @@ def test_overrides():
     assert config.value8 == ['abc', 'def', 'ghi']
 
 
+def test_overrides_boolean():
+    config = Config({}, {'value1': 'True',
+                         'value2': 'False',
+                         'value3': '1',
+                         'value4': '0',
+                         'value5': 'False'})
+    config.add('value1', None, 'env', [bool])
+    config.add('value2', None, 'env', [bool])
+    config.add('value3', None, 'env', [bool])
+    config.add('value4', None, 'env', [bool])
```

Comment:
We need to add `[bool]` to 4th parameter; `types` as a hint for type of the variable. It is required when it is hard to guess the type of default value.